### PR TITLE
Science Remap

### PR DESCRIPTION
**Pull request by Eyeveri**
Forum Thread for [Feedback](https://forums.aurorastation.org/topic/18075-science-remap/#comment-164087)

We 
Remapped - 
1.  The Entirety of the Science department on Deck Two.
2.  Circuits and RnD expanded.
3.  Science entrance moved.
4.  Xenobotany and Xenobiology received layout changes.
5. Xenobiology receives an extra, larger pen with some neat features. 
6.  Telescience was located to a different area.
7.  Exploratory Chemistry is now in its own secure area.
8.  AI Upload Foyer made smaller.
9.  AI Chamber altered slightly.
10.   Research Directors Office tweaked for QOL
11.  Science now has its own departmental entrance to a secure wing.
12.  Edits the Deck one ladder to the right by one tile
13. Machinists Workshop is now by Operations, which just makes more sense. 
14. Chapel had to be downsized slightly to fit the new workshop/Checkpoint alterations. 
15. Tool storage was relocated to be on the "port" side of the stargazer. Hopefully more people will use it now. 

MachinistRelocation
ScienceRemapWhole

### CODE REVIEW

**Comment by Eyeveri** [comment]
!wip

**Comment by Eyeveri** [comment]
@SleepyGemmy It is not only Research (RnD) that is remapped. Wouldn't this be "Science Remapped"?

File: html/changelogs/Eyeveri_Noob_ScienceRemap.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ReadThisNamePlz, 50%Noob
```
Comment (SleepyGemmy) [inline html/changelogs/Eyeveri_Noob_ScienceRemap.yml:30]:
```suggestion
author: ReadThisNamePlz, 50n00b
```
or equivalent (without the % symbol.)

File: html/changelogs/Eyeveri_Noob_ScienceRemap.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  - maptweak: "Circuits and RnD expanded."
+  - maptweak: "Science entrance moved."
+  - maptweak: "Xenobotany and Xenobiology received layout changes."
+  - maptweak: "Telescience was located to a different area."
+  - maptweak: "Exploratory Chemistry is now in its own secure area."
+  - maptweak: "AI Upload Foyer made smaller."
+  - maptweak: "AI Chamber altered slightly."
+  - maptweak: "Research Directors Office tweaked for QOL"
+  - maptweak: "Science now has its own departmental entrance to a secure wing."
+  - maptweak: "Edits the Deck one ladder to the right by one tile"
```
Comment (SleepyGemmy) [inline html/changelogs/Eyeveri_Noob_ScienceRemap.yml:51]:
```suggestion
  - maptweak: "Remapped the entirety of the research department on deck two."
```
one entry works fine.

File: maps/_common/areas/station/research.dm [hunk 3]
```diff
@@ -35,6 +35,10 @@
 	icon_state = "research"
 	lightswitch = TRUE
 
+/area/rnd/secure_hallway
+	name = "Research - Secure Hallway"
+	icon_state = "research"
+	lightswitch = TRUE
```
Comment (SleepyGemmy) [inline maps/_common/areas/station/research.dm:41]:
```suggestion
	lightswitch = TRUE

```

Comment (Eyeveri) [reply maps/_common/areas/station/research.dm:41]: What is the change here? I'm not seeing one.

Comment (SleepyGemmy) [reply maps/_common/areas/station/research.dm:41]: it adds a new line for spacing.

**Comment by Eyeveri** [comment]
!review

**Review by RyverStyx — APPROVED** [review]

**Review by RyverStyx — CHANGES_REQUESTED** [review]
Taking another look at this due to new changes... I also noticed you left in public access to the department. Remove this since so we can be consistent with all the other departments. The added lobby will mitigate the need for public access.

**Comment by Eyeveri** [comment]
Closing until an altered version is made.

**Pull request closed without merging**